ldk/javascript/src/whisper/types.ts

```diff
@@ -202,7 +202,7 @@ export enum IconSize {
 
 export type AutocompleteOption = {
   label: string;
-  value: string;
+  value: any;
```

Comment:
Not something to hold up the PR but we should add tests to make sure this actually works with anything other than strings, as far as I can tell we're not testing it?

Reply:
Looks like the build is failing at the lint stage because of this too, might want to disable eslint for this line for now?

Reply:
I did that and it's still failing :/